frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -80,10 +80,18 @@ export default function SideNavigation({
   toggleDrawer,
   currentUser,
 }: SideNavigationProps) {
-  const { numReviews, isNumReviewsLoading } = useGetNumReviews()
+  const [reviewCount, setReviewCount] = useState(0)
 
   const theme = useTheme()
 
+  useEffect(() => {
+    fetchReviewCount()
+  }, [])
+
+  async function fetchReviewCount() {
+    setReviewCount((await getReviewCount()).headers.get('x-count') as unknown as number)
```

Comment:
It doesn't look like there's any error handling around this. We should at least have some basic 200 status checks right?

Reply:
TODO added to handle this (and other instances like it) in another ticket

Reply:
Agreed we need accessible snackbar functionality to properly resolve this. This work will be spun out into a separate ticket to address this sort of problem across the entire app.